src/theme/page.php

```diff
@@ -26,9 +32,12 @@
     </article>
   <?php endwhile; ?>
 <?php else: ?>
-  Nothing here :(
+  <?php esc_html_e( 'Nothing here :(' ); ?>
```

Comment:
Translation string, see in the link below for more details:
https://codex.wordpress.org/Function_Reference/esc_html_e

Reply:
Ah I read that page but totally misunderstood. cool.

Reply:
But you are missing the text domain here. It should be `esc_html_e( 'Nothing here :(', 'wp-pwd-theme'  );`.

Reply:
Yes, but the text domain hasn't been defined.

Reply:
Normally the text domain is the theme slug which is `surmablog`. I did not realize what the theme name was before.

I feel that either it should be done correctly or not at all. The reason being is that if a translation was added it would not work till the text domain was defined.

The text domain in the header is only needed for the translation of the theme description.

Reply:
Ok, the correct is: 
`esc_html_e( 'Nothing here :(', 'surmablog' );`
I'll change it.
Thanks